wolfssl/wolfcrypt/asn_public.h

```diff
@@ -360,7 +362,9 @@ typedef struct CertName {
     CertOidField custom;
 #endif
 } CertName;
+#endif
```

Comment:
Please add the missing macro comment here.

Reply:
Done